package.json

```diff
@@ -146,18 +137,20 @@
     "@babel/preset-env": "^7.5.0",
     "@babel/preset-react": "^7.0.0",
     "@cerner/browserslist-config-terra": "^3.1.0",
-    "@cerner/eslint-config-terra": "^4.0.0",
+    "@cerner/eslint-config-terra": "^5.0.0",
+    "@cerner/stylelint-config-terra": "^4.0.0",
     "@cerner/terra-application": "cerner/terra-application#30f1e0a57f93ba119f08d4b95edd8ae4d597a866",
     "@cerner/terra-cli": "^1.1.0",
     "@cerner/terra-open-source-scripts": "^1.1.0",
     "@cerner/webpack-config-terra": "^1.0.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^24.8.0",
+    "browserslist-config-terra": "^2.0.0",
```

Comment:
😲

Reply:
Yes, i am not happy with this, I think the integration branch of terra app v2 is forcing me to add this. I will get it cleaned up in a future PR. I think we need to change terra app v2 to fix it.